modules/czechAdIdSystem.js

```diff
@@ -1,27 +1,27 @@
 /**
  * This module adds 'caid' to the User ID module
  * The {@link module:modules/userId} module is required
- * @module modules/cpexIdSystem
+ * @module modules/czechAdIdSystem
  * @requires module:modules/userId
  */
 
 import { submodule } from '../src/hook.js'
 import { getStorageManager } from '../src/storageManager.js'
 
 // Returns StorageManager
-export const storage = getStorageManager({ gvlid: 570, moduleName: 'cpexId' })
+export const storage = getStorageManager({ gvlid: 570, moduleName: 'czechAdId' })
 
 // Returns the id string from either cookie or localstorage
 const readId = () => { return storage.getCookie('czaid') || storage.getDataFromLocalStorage('czaid') }
 
 /** @type {Submodule} */
-export const cpexIdSubmodule = {
-  version: '0.0.5',
+export const czechAdIdSubmodule = {
```

Comment:
my strategy would be to export this as a function taking in the module name, and have a "dummy" `cpexIdSystyem.js` that re-registers under the old name until version 8. alternatively, you could delay this change until v8.